crates/bitcoind_rpc/src/lib.rs

```diff
@@ -43,29 +43,12 @@ pub struct Emitter<'c, C> {
 }
 
 impl<'c, C: bitcoincore_rpc::RpcApi> Emitter<'c, C> {
-    /// Construct a new [`Emitter`] with the given RPC `client` and `start_height`.
-    ///
-    /// `start_height` is the block height to start emitting blocks from.
-    pub fn from_height(client: &'c C, start_height: u32) -> Self {
+    /// TODO
```

Comment:
I wonder what we can do so that we don't forget to write this doc.

Reply:
Looks like docs here should be something like:
```suggestion
    /// Construct a new [`Emitter`] with the given RPC `client`, `last_cp` and `start_height`.
    ///
    /// `last_cp` is the check point used to find the latest block which is still part of the best chain.
    /// `start_height` is the block height to start emitting blocks from.
```